components/member-list-item/index.js

```diff
@@ -26,7 +26,7 @@ const PreviewMember = (props) => {
         </a>
       </Link>
 
-      <h2 className={classNames.name}>
+      <h2 className={classNames.nameOfPerson}>
         {data ? `${data['first_name']} ${data['last_name']}` : rdsId}
```

Comment:
default first name and last name like `${data['first_name']}||'--` and `${data['last_name']}||'--`

Reply:
@tejasberi I have added padding 10px 10px for classname as name  itself

Reply:
@Shihab-Duzno your comment is not related to my code change in this PR

Reply:
@harshi7016 we discussed about the class and agreed to use a unique name